webapi/appsettings.json

```
@@ -39,7 +39,7 @@
   "AIService": {
     "Type": "AzureOpenAI",
     "Endpoint": "", // ignored when AIService is "OpenAI"
-    // "Key": "",
+    "Key": "",
```

Don't tempt fate! :)

```suggestion
    // "Key": "",
```